extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/ColorSplasher.pushbutton/script.py

```diff
@@ -164,133 +142,136 @@ def Execute(self, uiapp):
                 list_max_X = []
                 list_y = []
-                #FilledRegionType
+                # FilledRegionType
                 filled_type=[]
-                all_types = FilteredElementCollector(new_doc).OfClass(FilledRegionType).ToElements()
+                all_types = DB.FilteredElementCollector(new_doc).OfClass(DB.FilledRegionType).ToElements()
                 for ty in all_types:
                     pattern = new_doc.GetElement(ty.ForegroundPatternId)
-                    if pattern != None:
+                    if pattern is not None:
                         if pattern.GetFillPattern().IsSolidFill:
                             if ty.ForegroundPatternColor.IsValid:
                                 filled_type.Add(ty)
                                 break
-                #Create Type if none is fill
+                # Create Type if none is fill
                 if len(filled_type) == 0 and len(all_types) > 0:
                     it = 1
-                    #Duplicate existing fillregiontype
+                    # Duplicate existing fillregiontype
                     while True:
                         try:
                             new_type = all_types[0].Duplicate("Fill Region " + str(it))
                             break
                         except:
-                            it +=1
+                            it += 1
                             if it == 100:
                                 break
-                    #Create pattern        
+                    # Create pattern
                     it = 1
                     while True:
                         try:
-                            new_pattern = FillPattern("Fill Pattern " + str(it), FillPatternTarget.Drafting, FillPatternHostOrientation.ToView, float(0), float(0.00001))
-                            new_elePat = FillPatternElement.Create(new_doc, new_pattern)
+                            new_pattern = DB.FillPattern("Fill Pattern " + str(it), DB.FillPatternTarget.Drafting, DB.FillPatternHostOrientation.ToView, float(0), float(0.00001))
+                            new_elePat = DB.FillPatternElement.Create(new_doc, new_pattern)
                             break
                         except:
-                            it +=1
+                            it += 1
                             if it == 100:
                                 break
-                    #Assign to type
+                    # Assign to type
                     new_type.ForegroundPatternId = new_elePat.Id
                     filled_type.Add(new_type)
-                #Create Text
+                # Create Text
                 for vw_item in wndw._listBox2.Items:
-                    punto = XYZ(0,0,0)
+                    punto = DB.XYZ(0, 0,  0)
                     index = wndw._listBox2.Items.IndexOf(vw_item)
-                    if index !=0:
-                        punto = XYZ(0,fin_coord_y,0)
+                    if index != 0:
```

Comment:
maybe `for index, vw_item in enumerate(wndw._listBox2.Items)` would work the same?